build.gradle

```diff
@@ -72,6 +72,8 @@ dependencies {
     compile 'com.jayway.jsonpath:json-path:2.2.0'
     compileOnly group: 'net.biville.florent', name: 'neo4j-sproc-compiler', version:'1.2'
 
+    compile 'com.novell.ldap:jldap:2009-10-07'
```

Comment:
this should be compileOnly + testOnly imho,
otherwise it's packaged into the jar.